account-management-app/src/main/webapp/WEB-INF/jspx/includes/storage-provider-form.jspx

```diff
@@ -1,190 +1,241 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <jsp:root
- version="2.0"
- xmlns="http://www.w3.org/1999/xhtml"
- xmlns:jsp="http://java.sun.com/JSP/Page"
- xmlns:c="http://java.sun.com/jsp/jstl/core"
- xmlns:fn="http://java.sun.com/jsp/jstl/functions"
- xmlns:tiles="http://tiles.apache.org/tags-tiles"
- xmlns:form="http://www.springframework.org/tags/form"
- xmlns:spring="http://www.springframework.org/tags">
-          <fieldset>
-            <legend>${param.storageProviderName}:  ${param.storageProviderType}</legend>
-              <form:hidden
-               path="${param.storageProvider}.id">
-              </form:hidden>
-            <ol>
-	            <li>
-	              <form:label
-	               cssErrorClass="error"
-	               path="${param.storageProvider}.username"><spring:message code="username"/></form:label>
-
-	              <form:input
-	               id="${param.storageProvider}username"
-	               cssErrorClass="error"
-	               path="${param.storageProvider}.username"
-	               onclick="same(this.form)"
-	               autofocus="true" />
+    version="2.0"
```

Comment:
The formatting changes here are a good thing (they are providing more consistent formatting), but they also obscure the functional changes. It would be great if the formatting changes could be made in an independent commit (or independent PR) so that it's easier to see what actual code is changing.